<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TemplateHome</class>
 <widget class="QMainWindow" name="TemplateHome">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Work Time Full</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icons/amp.png</normaloff>../icons/amp.png</iconset>
  </property>
  <property name="toolTip">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 75 16pt &quot;Arial&quot;;</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="ParamP">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Parâmetro Principal</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Periodo:*</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>20</width>
      <height>21</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>a</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="ParamC">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Parâmetro Complementar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Pessoa:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Setor:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonRegister">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>290</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Cadastrar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonView">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>290</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Visualizar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonCloseApp">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>290</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Fechar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputSectorId">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>190</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <stylestrategy>NoAntialias</stylestrategy>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;Arial&quot;;
text-transform: Uppercase;</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
    <property name="maxLength">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputPeopleId">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>230</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;Arial&quot;;
text-transform: Uppercase;</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
   </widget>
   <widget class="QDateEdit" name="inputLastDate">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>80</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;Arial&quot;;</string>
    </property>
    <property name="displayFormat">
     <string>d/M/yyyy</string>
    </property>
    <property name="date">
     <date>
      <year>2022</year>
      <month>12</month>
      <day>31</day>
     </date>
    </property>
   </widget>
   <widget class="QDateEdit" name="inputFirstDate">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;Arial&quot;;</string>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2022</year>
      <month>1</month>
      <day>1</day>
     </datetime>
    </property>
    <property name="displayFormat">
     <string>d/M/yyyy</string>
    </property>
    <property name="date">
     <date>
      <year>2022</year>
      <month>1</month>
      <day>1</day>
     </date>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSectorSelect">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>190</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonPeopleSelect">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>230</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputPeopleName">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>230</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;Arial&quot;;
text-transform: Uppercase;</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputSectorName">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>190</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;Arial&quot;;
text-transform: Uppercase;</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
